src/tools/parser/KomposTraceParser.java

```diff
@@ -0,0 +1,328 @@
+package tools.parser;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.nio.channels.FileChannel;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Scanner;
+import java.util.Set;
+
+import som.Output;
+import som.vm.VmSettings;
+import tools.debugger.entities.Marker;
+
+
+public class KomposTraceParser {
+
+  private static Set<Long> errorStrackTrace;
+
+  private static final int SOURCE_SECTION_SIZE = 8;
+
+  private enum TraceRecords {
+    ActivityCreation(11 + SOURCE_SECTION_SIZE),
```

Comment:
Don't we have such constants already somewhere?

Reply:
now using constant from TraceData.java